### ISSUE TITLE
no grid support in IE - properties with -ms prefixed not generated

### ISSUE THREAD

**Issue by mattwoodage** [emotion-js/emotion#1617]
How can i get -ms prefixes for grid properties to be generated?

Flex ones seem to work

`align-self: flex-end;`
generates
`-ms-flex-item-align: end;`
    
but 
`grid-row: 2;`
does not generate 
`-ms-grid-row: 2;`
    
grid has good support in IE.

what am i missing?

**Comment by Andarist** [comment]
If it doesnt work then seems like this is not supported by the parser we use: https://github.com/thysultan/stylis.js . Could you report it over there?

**Comment by mattwoodage** [comment]
they don't support it and don't seem to be replying to the open issues.

**Comment by grncdr** [comment]
I [shared a Stylis plugin](https://github.com/thysultan/stylis.js/issues/119#issuecomment-463539789) in their issue tracker last February, but the reality is that getting full grid support in IE11 is a *lot* of work and I didn't need it. 

The biggest hurdle is that auto-placement of grid items requires "whole stylesheet" knowledge, which Stylis does not make easily available. There are some informative articles out there about how Autoprefixer does it, and I did consider extending my plugin more, but I didn't need it.

For the product I work on, we looked at our traffic numbers and decided that IE11 support wasn't worth the time investment. If you (or anybody else) is in a situation where investing a few days is worth it, I'd be happy to dust off that code and do a Skype call or something to help you on the way to a more complete solution.
